app/config/MicroserviceAppConfig.scala

```diff
@@ -46,14 +46,24 @@ class MicroserviceAppConfig @Inject()(servicesConfig: ServicesConfig) {
       "Authorization" -> s"Bearer $ifToken"
     )
   }
-  val ifAuthHeaders1878 = {
+  val ifAuthHeaders1878: Seq[(String, String)] = {
     val ifEnvironment: String = loadConfig("microservice.services.if.environment")
     val ifToken: String = loadConfig("microservice.services.if.authorization-token-1878")
     Seq(
       "Environment" -> ifEnvironment,
       "Authorization" -> s"Bearer $ifToken"
     )
   }
+
+  val ifAuthHeaders1896: Seq[(String, String)] = {
```

Comment:
if we can add relevant unit tests for this change;
also, we can create a generic method after if we would like to extend this approach for any other API with own token etc;